ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionKorrektionsglasensStyrkaMessage.java

```diff
@@ -56,4 +58,9 @@ public static CertificateDataElement toCertificate(int index, CertificateTextPro
             )
             .build();
     }
+
+    private static Boolean setVisibility(Syn syn) {
+        final var korrektionsglasensStyrka = syn != null ? syn.getKorrektionsglasensStyrka() : null;
```

Comment:
For readability you could change this code to something like this:
 
        if (syn == null || syn.getKorrektionsglasensStyrka() == null) {
            return false;
        }
        return syn.getKorrektionsglasensStyrka();